packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -186,6 +190,9 @@ export class DeltaManager<TConnectionManager extends IConnectionManager>
     public get clientDetails() { return this.connectionManager.clientDetails; }
 
     public submit(type: MessageType, contents: any, batch = false, metadata?: any) {
+        if (this.opsCurrentlyProcessing > 0) {
+            this.close(new UsageError("Currently processing ops: Container closed"));
```

Comment:
Let's add better message here. I'd say "Making changes to data model is disallowed while processing ops."